nuitka/nodes/shapes/BuiltinTypeShapes.py

```diff
@@ -233,6 +241,10 @@ def getTypeName():
     def hasShapeSlotLen():
         return False
 
+    @staticmethod
```

Comment:
There is no such "slot". Any will work for `hasShapeSlotIter()`.

Reply:
Is it because there is no `__any__ ` slot or because it also take iterable as a input?